### PR TITLE
Quick silver sync config, updb and clear cache integration

### PR DESCRIPTION
**Pull request by enotick**

### CODE REVIEW

File: web/sites/default/settings.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+// $config['config_split.config_split.prod']['status'] = FALSE;
+// $config['config_split.config_split.dev']['status'] = FALSE;
+// Set environment indicator - foreground colour.
+$config['environment_indicator.indicator']['fg_color'] = '#FFFFFF';
+
+// Dev.
+if (
+  (isset($_ENV['AH_SITE_ENVIRONMENT']) && $_ENV['AH_SITE_ENVIRONMENT'] == 'dev') ||
+  (isset($_ENV['PANTHEON_ENVIRONMENT']) && $_ENV['PANTHEON_ENVIRONMENT'] == 'dev') ||
+  (getenv('ENVIRONMENT') == 'DEV')
```
Comment (RobLoach) [inline web/sites/default/settings.php:45]: We could likely extracts these environment checks to a function to improve readability. Something like....

``` php
if (current_environment() == 'dev')
```

Using a `switch` statement would also be easier to read.

Comment (enotick) [reply web/sites/default/settings.php:45]: @RobLoach definitely could. I took this way of doing it from BLT. They keep it fairly up to date and help to integrate with Pantheon and Acquia. When API changes happen they are easy to copy paste this way ;) as environment detection changes from time to time especially if we were to also include environment indicator (example is on config branch) and config split. However please let me know if you still would like me to refactor the code. I'd be open to do it.

**Comment by mikemccaffrey** [comment]
It looks like this pull request includes a lot of files that are generated by the scaffold and shouldn't be included in this repository. I will look evaluate the quicksilver part of it though.

**Comment by mikemccaffrey** [comment]
While the automatic configuration import would save time in many cases, it is a bit of a blunt approach that is not appropriate under some conditions. 
- Automatically config import on deployments to live is a great way to accidentally overwrite config changes that the clients have been made to their live site, so I would like the ability to review the config changes before import on live.
- If we are not automatically importing config on live, we should probably not do it on test either, since we want the process to be the same.
- The automatic config import may be most useful when pushing to dev. However, that is usually paired with a database refresh from live, which would immediately overwrite the database that you just ran the config import on.

I think that our first step might be using quicksilver to automatically create/refresh multidev environments, and get a feeling of how that works for us there before applying it to the master workflow.

**Pull request closed without merging**